app/src/main/res/layout/owner_detail_button.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="3dp"
+    android:padding="6dp"
+    android:layout_marginBottom="8dp"
+    android:id="@+id/owner_detail_button"
```

Comment:
we may delete this file?

Reply:
yes, I'll make sure it's deleted before it's merged